torchbox/torchbox.com
# Add animation for letter i in motif heading

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1396330906)

### Description of Changes Made
Added a special animation for the letter I in the motif heading component

### How to Test
- Navigate to any page with a motif heading, edit the heading to begin with I
- Ensure that the animation on the letter I is visible on page load

### Screenshots
[Loom screencast](https://www.loom.com/share/4cce8c56024c4f838fb209e73a3ef637?sid=86093361-968e-4c28-8721-0d6003fdae2c)

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility

- [] Not required

#### Sustainability


#### Pattern library
